contact.css

```diff
@@ -0,0 +1,129 @@
+
+.contact-section {
+  width: 375px;
+  height: 779px;
+  top: 4558px;
+  position: absolute;
+  background-color: #6070ff;
+  display: flex;
+  flex-direction: column;
+  background-image: url(images/contact_form_shapes-mobile.png);
+  background-repeat: no-repeat;
+  background-size: 222px 710px;
+  background-position: 160px 20px;
+  padding: 0 24px;
+  box-sizing: border-box;
+}
+
+.contact-header {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  padding-top: 103px;
+  gap: 12px;
+  align-self: center;
+}
+
+.contact-header h1 {
+  font-family: "Poppins", sans-serif;
+  font-style: bold;
+  font-weight: 700;
+  font-size: 40px;
+  color: #fff;
+  width: 327px;
+  height: 56px;
+  display: flex;
+  align-items: center;
+  padding-left: 50px;
+  box-sizing: border-box;
+}
+
+.contact-header p {
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  color: #ebebff;
+  width: 327px;
+  height: 102px;
+  align-self: center;
+  text-align: center;
+}
+
+.contact-form {
+  width: 327px;
+  margin-top: 87px;
+  align-self: center;
+}
+
+.contact-form-list {
+  display: flex;
+  list-style: none;
+  flex-direction: column;
+  gap: 18px;
+  padding-inline-start: 0;
+  height: fit-content;
+}
+
+.input-type-a {
+  width: 327px;
+  height: 48px;
+}
+
+.input-type-b {
+  width: 327px;
+  height: 170px;
+}
+
+#name {
+  width: 327px;
+  height: 48px;
+  border-radius: 8px;
+  border-color: white;
+  margin-bottom: 20px;
+}
+
+#mail {
+  width: 327px;
+  height: 48px;
+  border-radius: 8px;
+  border-color: white;
+  margin-bottom: 18px;
+}
+
+#message {
+  width: 327px;
+  height: 170px;
+  border-radius: 8px;
+  border-color: white;
+  margin-bottom: 22px;
+  resize: none;
+  outline: none;
+}
```

Comment:
- [ ] Excellent work 👍 💯  your `contact form` looks almost like the [ figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=MuQxvry65zLwhRdo-0). I suggest adding a little 🤏  bit of padding into the `input` fields to give a bit of space between the `placeholder text ` and the` borders`,  to make your design look like F[igma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=MuQxvry65zLwhRdo-0). As it is the [requirment](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) of the project to stick to the figma design template as much as possible

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).


Figma design:

<img width="273" alt="Screenshot 2023-02-28 at 10 12 23 PM" src="https://user-images.githubusercontent.com/44798044/221927027-ce24619e-fa42-4f9b-bfc8-6682f0d32b51.png">

Your design:

<img width="359" alt="Screenshot 2023-02-28 at 10 05 56 PM" src="https://user-images.githubusercontent.com/44798044/221926893-0586a318-ddb2-4bf4-87df-3e930d524946.png">